addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -855,7 +855,7 @@
                             <field name="user_id" widget="many2one_avatar_user" readonly="state != 'draft'"/>
                             <field name="accounting_date"
                                    groups="account.group_account_invoice,account.group_account_readonly"
-                                   invisible="state not in ['approve', 'post', 'done']"
+                                   invisible="state not in ['approve', 'post', 'done'] or payment_mode == 'company_account'"
```

Comment:
I haven't seen the request asking to hide it in the spec.
Did you discussed it with LABE?

I'm curious about the reason to hide it with this condition.